tests/codeception/acceptance/SubThemeCest.php

```diff
@@ -105,7 +105,6 @@ public function testMinimalSubtheme(AcceptanceTester $I) {
     $I->click('Set as default', 'a[title="Set Stanford Minimally Branded Subtheme as default theme"]');
     $I->amOnPage('/');
     $I->canSeeResponseCodeIs(200);
-    $I->dontSeeElement('.su-lockup__wordmark');
```

Comment:
This should be a good thing to test for. This would be a situation where we can test for the no-js solution. I haven't had a chance to review that yet though.

Reply:
Yes, I agree. It was failing since it is done through JS so I thought it had to be in a functional test instead.

Reply:
I'd prefer to keep this as an "acceptance" test and find a non-js solution for that part.

Reply:
Sounds good to me. I'd prefer that as well.